runWeathervane.pl

```diff
@@ -32,7 +34,9 @@ package Weathervane;
 			'version=s' => \$version,
 			'tmpDir=s' => \$tmpDir,
 			'script=s' => \$backgroundScript,
+			'fixedConfigsFile=s' => \$fixedConfigsFile,
```

Comment:
Does this open the possibility to achieve better scores with the same configurationsize by specifying a modified fixedConfigs file?

Reply:
Yes, but users can do that anyway by modifying the config file before building the runharness.  There is no way we keep users from "cheating" if they want to play with the configs.  Theoretically, the modified fixedConfigs file will end up in the output directory, but they could modify that also if they wanted.